usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -413,9 +418,20 @@ def add_arguments(self, parser):
             help="(OPTIONAL) detached_award_procurement_ids of FPDS transactions to load/reload from Broker",
         )
 
+        parser.add_argument(
+            "--id-file",
+            metavar="FILEPATH",
+            type=str,
+            help="A file containing only transaction IDs (detached_award_procurement_id) "
+            "to reload, one ID per line. Nonexistent IDs will be ignored.",
+        )
+
     @transaction.atomic
     def handle(self, *args, **options):
-        if options["detached_award_procurement_ids"] is not None:
-            self.load_specific_transactions(options["detached_award_procurement_ids"])
+        if any([options["detached_award_procurement_ids"], options["id_file"]]) is not None:
```

Comment:
`any()` is boolean, it will return True or False (not None)
For Python, it is considered best practice to leave off `is True` or `is not False` for boolean conditionals
```suggestion
        if any([options["detached_award_procurement_ids"], options["id_file"]]):
```

Reply:
https://docs.python.org/3.5/library/functions.html?highlight=any#any